mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown

```diff
@@ -36,7 +36,7 @@ resource "google_compute_instance_group_manager" "appserver" {
   zone               = "us-central1-a"
 
   version {
-    instance_template  = google_compute_instance_template.appserver.id
+    instance_template  = google_compute_instance_template.appserver.self_link_unique
```

Comment:
Also consider document that `self_link_unique` is preferred for `instance_template`.